readme.md

```diff
@@ -0,0 +1,161 @@
+# Remark for Visual Studio Code
```

Comment:
```suggestion
# vscode-remark
```

Should be package/extension name?

Reply:
I looked at what other extensions use, mainly the ones published by well known publishers and official extensions. `{tool} for Visual Studio Code` or something similar appears to be the most used level 1 heading in their readmes.

Reply:
Looked through the top 2 pages of the microsoft org on GH, by stars, searching for `vscode`.
I also find that there’s no consistency. A lot of random one-off weird things, but here’s the ones I mostly found:

1. `remark for Visual Studio Code` (https://github.com/microsoft/vscode-mssql#mssql-for-visual-studio-code, https://github.com/microsoft/vscode-docker#docker-for-visual-studio-code----, https://github.com/microsoft/vscode-postgresql#postgresql-for-visual-studio-code, https://github.com/microsoft/vscode-tools-for-ai#azure-machine-learning-for-visual-studio-code, https://github.com/microsoft/vscode-azurefunctions#azure-functions-for-visual-studio-code)
2. `remark for VS Code` (https://github.com/microsoft/vscode-sublime-keybindings, https://github.com/microsoft/vscode-cosmosdb, https://github.com/microsoft/vscode-atom-keybindings)
3. `vscode-remark` (https://github.com/microsoft/vscode-css-languageservice#vscode-css-languageservice, https://github.com/microsoft/vscode-json-languageservice#vscode-json-languageservice)

I’m open to 1. Though, as 3. is used sometimes by them and everywhere by us, that has my preference.

Reply:
I used 1, as the references for option 3 don’t link to VSCode plugins.